services/david/david.service.js

```diff
@@ -25,30 +25,10 @@ module.exports = class David extends LegacyService {
     return [
       {
         title: 'David',
-        pattern: ':user/:repo',
+        pattern: ':kind(dev|optional|peer)?:user/:repo',
         namedParams: { user: 'expressjs', repo: 'express' },
```

Comment:
The examples were one of the things I was wondering about. Will we need to have multiple entry for each option, or only in certain cases, or something else?

Reply:
I think we can start with all the cases where we have patterns which are identical, other than a multiple-choice option. The ones that come to mind are the download badges, the `v`/`vpre` badges, and a few with parameters called `:which`, like GitHub pull request detail.

It may turn out that we want to define the terms in the UI, though to start, that could be done in documentation.

Reply:
Is that question mark supposed to be a `/`?

Reply:
Ah, the `?` is supposed to be there but it's supposed to have a `/` after it.